scripts/gulp-tasks.js

```diff
@@ -0,0 +1,166 @@
+"use strict";
+
+/**
+ * This file contains the Gulp tasks for babel-standalone. Note that
+ * babel-standalone is compiled using Webpack, and performs its own Babel
+ * compilation of all the JavaScript files. This is because it targets web
+ * browsers, so more transforms are needed than the regular Babel builds that
+ * only target Node.js.
+ *
+ * The tasks in this file are designed to be reusable, so that they can be used
+ * to make standalone builds of other Babel plugins/presets (such as babel-minify)
+ */
+
+const path = require("path");
+const pump = require("pump");
+const rename = require("gulp-rename");
+const RootMostResolvePlugin = require("webpack-dependency-suite")
+  .RootMostResolvePlugin;
+const webpack = require("webpack");
+const webpackStream = require("webpack-stream");
+const uglify = require("gulp-uglify");
+
+const standalonePackagePlugins = {
+  babelPresetEnv: [
+    new webpack.NormalModuleReplacementPlugin(
+      /\.\/available-plugins/,
+      require.resolve(
+        path.join(
+          __dirname,
+          "../experimental/babel-preset-env-standalone/src/available-plugins"
+        )
+      )
+    ),
+    new webpack.NormalModuleReplacementPlugin(
+      /caniuse-lite\/data\/regions\/.+/,
+      require.resolve(
+        path.join(
+          __dirname,
+          "../experimental/babel-preset-env-standalone/src/caniuse-lite-regions"
+        )
+      )
+    ),
+  ],
+};
+
+function webpackBuild(opts) {
+  const plugins = standalonePackagePlugins[opts.library] || [];
+  let babelVersion = require("../packages/babel-core/package.json").version;
+  let version = opts.version || babelVersion;
+  // If this build is part of a pull request, include the pull request number in
+  // the version number.
+  if (process.env.CIRCLE_PR_NUMBER) {
+    const prVersion = "+pr." + process.env.CIRCLE_PR_NUMBER;
+    babelVersion += prVersion;
+    version += prVersion;
+  }
+
+  const config = {
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
+          include: /node_modules/,
+          loader: "babel-loader",
+          options: {
+            // Some of the node_modules may have their own "babel" section in
+            // their project.json (or a ".babelrc" file). We need to ignore
+            // those as we're using our own Babel options.
+            babelrc: false,
+            presets: ["env"],
```

Comment:
I just realised that babel-standalone is being built with the `env` preset. How does it determine which browsers to support? I wonder if this broke it in older browsers...

Reply:
@Daniel15 env without any targets passed in acts the same as babel-preset-latest